### PR TITLE
Tutorial updates

### PR DESCRIPTION
**Pull request by sserita**
This PR is only a partial completion of #317, but I'm including the current bugfixes in the next minor release because there is no reason to wait on all the updates and these include some high-visibility/-utility updates.

Included in this PR are the following items from #317:

- Experiment design construction tutorials (for using/explaining new germ, fiducial and FPR tools). Maybe a new tutorial on using the fisher information to evaluate experiment designs
- Parity POVM example
- Specifying reduced error generator models using new coefficient block interface
- 00-Protocols tutorial issues
- 01-Esssential-Objects tutorial issues
- 02-Using-Essential-Objects tutorial issues
- CustomOperator tutorial issues

This PR also fixes the following related issues:

- Fixing the ModelTesting-functions tutorial in #247
- Warning JupyterLab incompatibility in #328

### CODE REVIEW

**Comment by sserita** [comment]
Woot I think this is our first PR that is pulling in code owners outside of the dev team, which means that @coreyostrove you are off the hook (unless you also wanted to look).

@kmrudin and @pcwysoc: I know this is probably new for you. I'm happy to hop on a Teams call to help with the process. But the TL;DR is:

1. Go over to the Files Changed tab.
2. Any file with a blue shield in the top left is one that you are a code owner for. Look over the changes. Red is a removed line, green is an added line. For a changed line, you'll see a red followed immediately by a green, with the thing that is changed being highlighted.
4. If you want to leave a comment, you can click on the line number and leave your comment.
5. When you are done with a file, you can click on the Viewed box in the top right and move to the next one.
6. When you have done all the files, click Review Changes in the top right above the files and either Approve if all looks good, Request changes if something looks bad, and Comment if you have questions but are not explicitly approving/rejecting.

If you come across a file that you are not a code owner for, feel free to skip it. If you come across a file that you ARE a code owner for and don't feel comfortable reviewing it, please comment that in the review - I can get someone else to review and then alter code owners appropriately.

@kmrudin Unfortunately, Jupyter notebooks are particularly annoying to review here. For the typo updates, this should be relatively OK, but for the ModelNoise.ipynb and CustomMeasurement.ipynb, you may find it helpful to look at the tutorial notebooks [here](https://github.com/sandialabs/pyGSTi/tree/bugfix-example-updates/jupyter_notebooks). For ModelNoise, the "Reduced error models" section is new, and the entire notebook is new for CustomMeasurement.

File: jupyter_notebooks/Tutorials/reporting/ReportGeneration.ipynb
Comment (pcwysoc) [inline jupyter_notebooks/Tutorials/reporting/ReportGeneration.ipynb:1]: Looks good! Glad to see a warning about the JupyterLab incompatibility. The link to the issue appears to work, as does the color.

**Review by pcwysoc — APPROVED** [review]
The short bit I'm on the hook for looks good to me.

File: jupyter_notebooks/Tutorials/01-Essential-Objects.ipynb
Comment (kmrudin) [inline jupyter_notebooks/Tutorials/01-Essential-Objects.ipynb:1]: We should also mention here that there is an option to not store 0-counts in the first place.

**Review by kmrudin — CHANGES_REQUESTED** [review]
Just a few minor changes; discussed with Stefan.

File: jupyter_notebooks/Tutorials/01-Essential-Objects.ipynb
Comment (sserita) [reply jupyter_notebooks/Tutorials/01-Essential-Objects.ipynb:1]: Done!

**Review by kmrudin — APPROVED** [review]

**Pull request merged**
